README.md

```diff
@@ -10,150 +10,148 @@
 
 A GitHub plugin for Mattermost. Supports GitHub SaaS and Enterprise versions.
 
-![GitHub plugin screenshot](https://user-images.githubusercontent.com/13119842/54380268-6adab180-4661-11e9-8470-a9c615c00041.png)
+## Table of Contents
 
-## Features
+ - [Audience](#audience)
+ - [License](#license)
+ - [About the GitHub Plugin](#about-the-github-plugin)
+ - [Before You Start](#before-you-start)
+ - [Configuration](#configuration)
+ - [Using the Plugin](#using-the-plugin)
+ - [Onboarding Your Users](#onboarding-your-users)
+ - [Slash Commands](#slash-commands)
+ - [Frequently Asked Questions](#frequently-asked-questions)
+ - [Development](#development)
 
-After your System Admin has [configured the GitHub plugin](#Configuration), run `/github connect` in a Mattermost channel to connect your Mattermost and GitHub accounts.
+![GitHub plugin screenshot](assets/github_mattermost.png)
```

Comment:
Files in`assets` as bundled with the plugin release which is not needed. Could you please put the file into `images` similar to how we do it in https://github.com/mattermost/mattermost-plugin-jira?

Reply:
I don't see an ``images`` directory in this repo - wondering if the Jira repo has one because of it being on Gitbook?

Reply:
Yes, you need to create an `image` directory. The assets for GitBook are stored in https://github.com/mattermost/mattermost-plugin-jira/tree/master/docs/.gitbook/assets